packages/expo-notifications/README.md

```diff
@@ -723,6 +724,40 @@ Cancels all scheduled notifications.
 
 A `Promise` resolving once all the scheduled notifications are successfully cancelled or if there are no scheduled notifications.
 
+### `getNextTriggerDateAsync(trigger: NotificationTriggerInput): Promise<number | null>`
+
+Allows you to check what will be the next trigger date for given notification trigger input.
+
+#### Arguments
+
+The notification trigger you would like to check next trigger date for.
+
+> **Note:** Only calendar-based notification triggers are supported, trying to get trigger date for triggers of other types will result in `Promise` rejection with code `ERR_NOTIFICATIONS_INVALID_CALENDAR_TRIGGER`.
+
+#### Returns
+
+ If return value is `null`, the notification wouldn't be triggered. If return value is a number, it is Unix timestamp at which the notification would be triggered (to interpret it as a JS `Date` you'll need to multiply the number by 1000).
```

Comment:
```suggestion
If the return value is `null`, the notification won't be triggered. Otherwise, the return value is the Unix timestamp at which the notification will be triggered (to interpret it as a JS `Date` you'll need to multiply the number by 1000).
```